x-pack/plugins/alerting/server/alerts_client/alerts_client.ts

```diff
@@ -813,4 +823,17 @@ export class AlertsClient<
   public isUsingDataStreams(): boolean {
     return this._isUsingDataStreams;
   }
+
+  private throwIfHasClusterBlockException(response: BulkResponse) {
+    response.items.forEach((item) => {
+      const op = item.create || item.index || item.update || item.delete;
+      if (op?.error && op.error.type === CLUSTER_BLOCK_EXCEPTION) {
+        throw new ErrorWithType({
+          message: op!.error!.reason,
+          type: CLUSTER_BLOCK_EXCEPTION,
+          stack: op.error.stack_trace,
```

Comment:
nit: op?.error?.stack_trace

Reply:
it is already in a if case that checks `op?.error`